docs/siem/detections/alerts-ui-manage.asciidoc

```diff
@@ -0,0 +1,98 @@
+[[alerts-ui-manage]]
+[role="xpack"]
+== Managing detection alerts
+
+The Detections page displays all alerts. From the Alerts table, you can change
+an alert's status, and start investigating and analyzing alerts in Timeline.
+
+TIP: From Timeline, you can <<cases-ui-open, create cases>> to track issues and
+share information with colleagues.
+
+To view detection alerts created by a specific rule, you can:
+
+* Filter for a specific rule in the KQL bar (for example,
+`signal.rule.name :"SSH (Secure Shell) from the Internet"`).
+* View detection alerts in the *Rule details* page (click
+*Manage detection rules* -> rule name in the *All rules* table).
+
+NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the 
+*Detections* and *Rule details* pages, and in Timeline when either `All` or
+`Detection alerts` is selected. 
+
+TIP: Use the icons in the upper left corner of the Alerts table to customize
+displayed columns and row renderers, and view the table in full screen mode.
+
+[float]
+[[detection-alert-status]]
+=== Change alert statuses
+
+You can set an alert's status to indicate whether it needs to be investigated
+(`Open`), is under active investigation (`In progress`), or resolved
+(`Closed`). By default, the Alerts table displays open alerts. To view alerts
+with other statuses, click *In progress* or *Closed*.
+
+To change alert statuses, either:
+
+* In the alert's row, click the *more options* icon, and then select the
+required status (*Mark in progress*, *Close alert*, or *Open alert*).
+* In the Alerts table, select all the alerts you want to change, and then select
+*Take action* -> *Close selected*, *Open selected*, or *Mark in progress*.
+
+[float]
+[[signals-to-timelines]]
+=== Send alerts to Timeline
+
+To send an alert in Timeline, click the *Investigate in timeline* icon.
```

Comment:
```suggestion
To send an alert to Timeline, click the *Investigate in timeline* icon.
```

Or some variant? _To view an alert in Timeline_?

Reply:
One extra thing that might be worth mentioning here is that for `Threshold Rules`, the `Investigate in timeline` action will also include the corresponding events that were bucketed to meet the specified threshold. So if you've got a threshold rule looking for >20 failed logins over the last interval, the `investigate in timeline` action for those detection alerts will send not only the detection alert, but also all the failed auth events as well (which can be super useful! :).

(Feel free to ignore if you cover this in the other Threshold Rules docs)